APID: Doc changes for 2.13.0 (#133)
mulesoft/docs-design-center: modules/ROOT/pages/design-create-publish-api-raml-editor.adoc

== Procedure

. Open Design Center from the homepage of Anypoint Platform.
. Create an API-specification project that you can work in.
 .. On the *Projects* page in Design Center, click *Create* and then click *Create API Specification*.

>  .. In the *New Specification* section of the dialog that is opened, name your project. You can change the name (but not the extension) later, if you want to.

Reviewer: ```suggestion
 .. In *New Specification*, name your project. 
 You can change the name (but not the extension) later, if you want to.
```

Author: @kbostrom , I disagree with rendering the text vague.

>  .. Select the option *Text editor*.

Reviewer: ```suggestion
 .. Select *Text editor*.
```

 .. Choose whether the stub specification file created in the project should be in RAML or OAS. The default is to create a RAML 1.0 file; however, you can change the file to RAML 0.8 or OAS after the project is created. In fact, projects for creating API specifications can contain a mix of RAML and OAS files.

+
*Result:* The text editor opens. The editor is divided into three panels:

* The left panel lists the files in your project.
+

> The stub specification file is listed and set as the root file of the project. You can change the name and extension of this file by right-clicking the three dots to the right of the name and selecting *Rename*.

Author: Fixed.

+
For more information, view the topic "Simulate Calls to an API", which is linked to from the *See also* section at the end of this topic.

> Export your API project or specification:: At any time, you can export your project as a `.zip` file. Click the gear icon in the top-right corner of the text editor and select *Download Project*.

Reviewer: ```suggestion
Export your API project or specification:: At any time, you can export your project as a ZIP file. Click the gear icon in the text editor and select *Download Project*.
```

== What to Do Next

If you are satisfied with your API specification, you can publish it to Exchange. See xref:design-publish.adoc[Publish an API Specification].

== See Also
// 181129: Deleting the file that this xref links to. * xref:design-export-files.adoc[Export Files from an API Project]
* xref:design-import-files.adoc[Import Files into an API Project]
* xref:design-add-api-dependency.adoc[Add a RAML API Fragment to an API-Specification Project as a Dependency]
* xref:design-mocking-service.adoc[Simulate Calls to an API]
* xref:design-change-root-file.adoc[CRoot Files in Projects in the Text Editor]
* xref:design-publish.adoc[Publish an API Specification]
* xref:design-rename-project.adoc[Rename an Open Project]
* xref:design-delete-project.adoc[Delete an Open Project]

////
Text to add at line 17 when the GitHub integration is no longer "dark":
 .. Choose whether to save your project in Design Center or in a repository in GitHub.
 +

>  * If you save your project in Design Center, you or other people in your organization can fork your project in Design Center, as described in xref:design-branching.adoc[Fork a Project in the Text Editor].

Reviewer: ```suggestion
 * If you save your project in Design Center, you or others in your organization can fork your project in Design Center, as described in xref:design-branching.adoc[Fork a Project in the Text Editor].
```
